### PR TITLE
Converted dev scripts to ESM modules

### PR DESCRIPTION
**Pull request by KolCrooks**
I converted the dev scripts to ESM modules as talked about in #701.

### CODE REVIEW

**Comment by MarcusNotheis** [comment]
Hey @KolCrooks,
could you update your branch with the latest master version (or give me write access to that branch in case there are more commits that have to be merged)? Then we can get this into our codebase.
Thanks!

File: shared/tests/serializer/content-loader-serializer.js [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
+  value = value.replace(/url\((.*)\)/, 'STYLE-URL');
 
-    return serialize(value);
+  if (/-diff$/.test(value)) {
+    value = 'DIFF_ID';
   }
-};
+
+  return serialize(value);
+}
```
Comment (MarcusNotheis) [inline shared/tests/serializer/content-loader-serializer.js:17]: Could you please reset those changes and rename the file to use the `.cjs` extension?

File: shared/tests/serializer/jss-snapshot-serializer.js [hunk 2]
```diff
@@ -52,23 +52,21 @@ const replaceJssClassNames = (elements) => {
...
-    replaceJssClassNames(elements);
+  // remove the non-deterministic part from the JSS class names
+  // to keep the snapshots repeatable
+  replaceJssClassNames(elements);
 
-    return serialize(value);
-  }
-};
+  return serialize(value);
+}
```
Comment (MarcusNotheis) [inline shared/tests/serializer/jss-snapshot-serializer.js:72]: Could you please reset those changes and rename the file to use the `.cjs` extension?

**Review by MarcusNotheis — CHANGES_REQUESTED** [review]
It looks like jest serializers have to be in `cjs` format, so could you please revert the changes in both serializers (marked as PR comment) and rename them to use the `.cjs` file extension?
Then you have to update the imports in `config/jestsetup.ts` to match the new `.cjs` file extension:
```js
import contentLoaderSerializer from '@shared/tests/serializer/content-loader-serializer.cjs';
import jssSerializer from '@shared/tests/serializer/jss-snapshot-serializer.cjs';
```
Other than that everything looks good and we can merge this PR as soon as all CI jobs are passing.

Thanks!

**Review by MarcusNotheis — APPROVED** [review]

**Comment by KolCrooks** [comment]
@MarcusNotheis I'm sorry that I haven't been able to work on this / not giving a heads up. I got hit with a ton of work and midterms from uni so It has been tough for me to find time to do things like this.

**Comment by MarcusNotheis** [comment]
Key @KolCrooks,
No worries, all good - as it was only a very little change that was pending I just added and merged it :) thanks for your work on that PR and good luck for your midterms :)

Best regards,
Marcus

**Pull request merged**